__tests__/kata1.fizzBuzz.test.js

```diff
@@ -2,18 +2,26 @@ const { fizzBuzz } = require('../src');
 
 describe('fizzBuzz', () => {
   it('returns Fizz when passed a multiple of 3', () => {
-
+    expect (fizzBuzz(9)).toBe('Fizz')
```

Comment:
Have spaces before parentheses when **declaring** functions, but not when **calling** functions:

```js
expect(fizzBuzz(9)).toBe('Fizz')
```